.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 .rpt2_cache
+.idea
```

Comment:
Please delete this line. `.idea` directory is required only InteliJ users.

Reply:
Sure, I deleted `.idea` and updated info in [README.md](https://github.com/Jungwoo-An/react-editor-js#-api).
Thank you for reviewing this PR.